src/components/AccessibilityPolicyForm/index.jsx

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Button from '@edx/paragon/src/Button';
+import InputText from '@edx/paragon/src/InputText';
+import StatusAlert from '@edx/paragon/src/StatusAlert';
+import TextArea from '@edx/paragon/src/TextArea';
+
+import { clearAccessibilityStatus, submitAccessibilityForm } from '../../data/actions/accessibility';
+import { accessibilityActions } from '../../data/constants/actionTypes';
+import styles from './AccessibilityPolicyForm.scss';
+
+export class AccessibilityPolicyForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      formEmail: '',
```

Comment:
Naming nit: this class has `Form` in the title - might it make sense to remove `form` from the names of the internal variables?

Reply:
This makes sense and is easy enough to clean up for readability - I am going to make `formEmail` something like `submitterEmail` though because `email` is a prop that is already passed in to handle the a11y email address (I'm going to change that to be `accessibilityEmail` for additional clarity)